information-mediator/docs/main.md

```diff
@@ -0,0 +1,473 @@
+ $ sed -i 's/${APPLICATION_ENABLED}/'"$APPLICATION_ENABLED"'/g' val.yaml
+ $ sed -i 's/${ECR_CS_REPO}/'"$ECR_CS_REPO"'/g' val.yaml
+ $ sed -i 's/${ECR_SS_REPO}/'"$ECR_SS_REPO"'/g' val.yaml
+ $ sed -i 's/${ECR_CS_IMAGE_NAME}/'"$ECR_CS_IMAGE_NAME"'/g' val.yaml
+ $ sed -i 's/${ECR_SSM_IMAGE_NAME}/'"$ECR_SSM_IMAGE_NAME"'/g' val.yaml
+ $ sed -i 's/${ECR_SSC_IMAGE_NAME}/'"$ECR_SSC_IMAGE_NAME"'/g' val.yaml
+ $ sed -i 's/${ECR_SSP_IMAGE_NAME}/'"$ECR_SSP_IMAGE_NAME"'/g' val.yaml
+ $ sed -i 's/${CS_DB_HOST}/'"$CS_DB_HOST"'/g' val.yaml
+ $ sed -i 's/${CS_DB_PORT}/'"$CS_DB_PORT"'/g' val.yaml
+ $ sed -i 's/${CS_DB_ADMIN_USER}/'"$CS_DB_ADMIN_USER"'/g' val.yaml
+ $ sed -i 's/${CS_DB_ADMIN_PASS}/'"$CS_DB_ADMIN_PASS"'/g' val.yaml
+ $ sed -i 's/${CS_DB_UI_PASS}/'"$CS_DB_UI_PASS"'/g' val.yaml
+ $ sed -i 's/${SSC_DB_HOST}/'"$SSC_DB_HOST"'/g' val.yaml
+ $ sed -i 's/${SSC_DB_PORT}/'"$SSC_DB_PORT"'/g' val.yaml
+ $ sed -i 's/${SSC_DB_ADMIN_USER}/'"$SSC_DB_ADMIN_USER"'/g' val.yaml
+ $ sed -i 's/${SSC_DB_ADMIN_PASS}/'"$SSC_DB_ADMIN_PASS"'/g' val.yaml
+ $ sed -i 's/${SSC_DB_SERVERCONF_ADMIN_PASS}/'"$SSC_DB_SERVERCONF_ADMIN_PASS"'/g' val.yaml
+ $ sed -i 's/${SSC_DB_MESSAGELOG_ADMIN_PASS}/'"$SSC_DB_MESSAGELOG_ADMIN_PASS"'/g' val.yaml
+ $ sed -i 's/${SSC_DB_SERVERCONF_PASS}/'"$SSC_DB_SERVERCONF_PASS"'/g' val.yaml
+ $ sed -i 's/${SSC_DB_MESSAGELOG_PASS}/'"$SSC_DB_MESSAGELOG_PASS"'/g' val.yaml
+ $ sed -i 's/${SSM_DB_HOST}/'"$SSM_DB_HOST"'/g' val.yaml
+ $ sed -i 's/${SSM_DB_PORT}/'"$SSM_DB_PORT"'/g' val.yaml
+ $ sed -i 's/${SSM_DB_ADMIN_USER}/'"$SSM_DB_ADMIN_USER"'/g' val.yaml
+ $ sed -i 's/${SSM_DB_ADMIN_PASS}/'"$SSM_DB_ADMIN_PASS"'/g' val.yaml
+ $ sed -i 's/${SSM_DB_SERVERCONF_ADMIN_PASS}/'"$SSM_DB_SERVERCONF_ADMIN_PASS"'/g' val.yaml
+ $ sed -i 's/${SSM_DB_MESSAGELOG_ADMIN_PASS}/'"$SSM_DB_MESSAGELOG_ADMIN_PASS"'/g' val.yaml
+ $ sed -i 's/${SSM_DB_SERVERCONF_PASS}/'"$SSM_DB_SERVERCONF_PASS"'/g' val.yaml
+ $ sed -i 's/${SSM_DB_MESSAGELOG_PASS}/'"$SSM_DB_MESSAGELOG_PASS"'/g' val.yaml
+ $ sed -i 's/${SSP_DB_HOST}/'"$SSP_DB_HOST"'/g' val.yaml
+ $ sed -i 's/${SSP_DB_PORT}/'"$SSP_DB_PORT"'/g' val.yaml
+ $ sed -i 's/${SSP_DB_ADMIN_USER}/'"$SSP_DB_ADMIN_USER"'/g' val.yaml
+ $ sed -i 's/${SSP_DB_ADMIN_PASS}/'"$SSP_DB_ADMIN_PASS"'/g' val.yaml
+ $ sed -i 's/${SSP_DB_SERVERCONF_ADMIN_PASS}/'"$SSP_DB_SERVERCONF_ADMIN_PASS"'/g' val.yaml
+ $ sed -i 's/${SSP_DB_MESSAGELOG_ADMIN_PASS}/'"$SSP_DB_MESSAGELOG_ADMIN_PASS"'/g' val.yaml
+ $ sed -i 's/${SSP_DB_SERVERCONF_PASS}/'"$SSP_DB_SERVERCONF_PASS"'/g' val.yaml
+ $ sed -i 's/${SSP_DB_MESSAGELOG_PASS}/'"$SSP_DB_MESSAGELOG_PASS"'/g' val.yaml 
+```
+
+Finally, that temporary values file is used to install IM to Kubernetes Cluster:
+``` bash
+ $ helm upgrade --install information-mediator ./information-mediator/ -f val.yaml 
+```
+
+11. Structure of Helm charts
+
+The "information-mediator" chart contains the following sub-charts:
+
+* A chart for [X-Road Central Server](information_mediator/charts/xroad-cs)
```

Comment:
These links do not work anymore as you have moved the documentation.

Reply:
Fix c84a4f112a5053772e60042ca260d90705e09962